long-live-pomf.md

```diff
@@ -17,7 +17,7 @@
 |                            |                  | https://github.com/kimoi/madokami.com         |                  |
 |                            |                  | https://github.com/bohrmeista/1338            |                  |
 |                            |                  | https://github.com/nya/cpomf                  |                  |
-| https://nup.pw/            | Up               |                                               |              150 | JS-free
+| https://nup.pw/            | Down (01/Oct-17) |                                               |              150 | JS-free
```

Comment:
You used `/` instead of `-` before, why the change?

Reply:
A-accident?